File: 프로그래머스/lv1/12935. 제일 작은 수 제거하기/제일 작은 수 제거하기.java
class Solution {
    public int[] solution(int[] arr) {
        if(arr.length<=1) return new int[]{-1};
        int min= arr[0];
        int[] answer = new int[arr.length-1];
        for(int i : arr){ min = Math.min(i,min);}
        int idx = 0;
        for(int i =0; i < arr.length;i++){
            if(arr[i]==min) {continue;}
            else{answer[idx] = arr[i]; idx++;}
        }
        return answer;
}
}

File: 프로그래머스/lv1/12935. 제일 작은 수 제거하기/제일 작은 수 제거하기.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

vector<int> solution(vector<int> arr) {
    if(arr.size()<=1) return {-1};
    
    arr.erase(min_element(arr.begin(),arr.end()));
    return arr;
}

